hs_geographic_feature_resource/tests/test_geo_feature.py

```diff
@@ -0,0 +1,580 @@
+__author__ = 'drew'
+
+from unittest import TestCase
+from django.contrib.contenttypes.models import ContentType
+from hs_core.hydroshare import utils, users, resource
+from hs_core.models import GenericResource, Creator, Contributor, CoreMetaData, \
+    Coverage, Rights, Title, Language, Publisher, Identifier, \
+    Type, Subject, Description, Date, Format, Relation, Source
+
+from hs_geographic_feature_resource.models import *
+from hs_geographic_feature_resource.receivers import *
+from django.contrib.auth.models import Group, User
+from hs_core import hydroshare
+from django.core.files.uploadedfile import UploadedFile
+from django.http import HttpRequest, QueryDict
+
+class TestGeoFeatureMetadata(TestCase):
+    def setUp(self):
+        try:
+            #self.user = User.objects.create_user('user1', email='user1@nowhere.com')
+            self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+            self.user = hydroshare.create_account(
+                'zhiyu.li@byu.edu',
+                username='drew',
+                first_name='Zhiyu',
+                last_name='Li',
+                superuser=False,
+                groups=[self.group]
+            )
+            self.allowance=0.00001
+        except:
+            self.tearDown()
+            self.user = User.objects.create_user('user1', email='user1@nowhere.com')
```

Comment:
Swallowing errors in tests is a good way for failures to go unseen. Please do not do this.